src/openzeppelin/security/safemath.cairo

```diff
@@ -7,109 +7,114 @@ from starkware.cairo.common.cairo_builtins import HashBuiltin, SignatureBuiltin
 from starkware.cairo.common.bool import TRUE, FALSE
 from starkware.cairo.common.math import assert_not_zero
 from starkware.cairo.common.uint256 import (
-    Uint256, uint256_check, uint256_add, uint256_sub, uint256_mul, 
+    Uint256, uint256_check, uint256_add, uint256_sub, uint256_mul,
     uint256_unsigned_div_rem, uint256_le, uint256_lt, uint256_eq
 )
 
-# Adds two integers. 
-# Reverts if the sum overflows.
-func uint256_checked_add{
-        syscall_ptr: felt*, 
-        pedersen_ptr: HashBuiltin*, 
-        range_check_ptr
-    } (a: Uint256, b: Uint256) -> (c: Uint256):
-    uint256_check(a)
-    uint256_check(b)
-    let (c: Uint256, is_overflow) = uint256_add(a, b)
-    with_attr error_message("Safemath: addition overflow"):
-        assert is_overflow = FALSE
+namespace SafeMath:
+
+    # Adds two integers. 
+    # Reverts if the sum overflows.
+    func uint256_checked_add{
```

Comment:
```suggestion
    func uint256_add{
```
now that there's a namespace I think we should drop some redundant info from the name. we could even drop uint256 if we call it SafeUint256 or something like that

Reply:
I originally switched it to `uint256_add` (and the rest), but it conflicts with the Cairo Uint256 imports which is why I kept the long names. Changing the dir/namespace to SafeUint256 is a good idea and should fix that